R/cell_suppression.R

```diff
@@ -134,7 +135,7 @@ render_cell_suppression.default <- function(
   if (is.logical(x)) x <- factor(x, levels = c(T, F), labels = c("Yes", "No"))
 
   if (is.factor(x) || is.character(x)) {
-    r <- do.call(render.categorical, list(x = x))
+    r <- do.call(render.categorical, c(list(x = x), list(...)))
```

Comment:
No functional difference but this would be more readable as `list(x = x, ...)` instead of `c(list(x = x), list(...))`

Reply:
Thanks for reviewing @vaakesan-SMH! Ah yes, good point. I fixed this now and will go ahead and merge.